sympy/solvers/tests/test_solvers.py

```diff
@@ -1605,14 +1605,47 @@ def test_high_order_roots():
 
 
 def test_minsolve_linear_system():
+    assert solve([x + y - 5, 2*x - y - 1], quick=True) == {x: 2, y: 3}
+    assert solve([x + y - 5, 2*x - y - 1], quick=False) == {x: 2, y: 3}
     def count(dic):
         return len([x for x in dic.values() if x == 0])
-    assert count(solve([x + y + z, y + z + a + t], particular=True, quick=True)) \
+    assert count(solve([x + y + z, y + z + a + t], quick=True)) \
```

Comment:
The quick=True flag is already quite misleading I think but making it imply particular=True doesn't help.

Reply:
Setting `quick=True` has no meaning outside of the `particular=True` context. Do you think it would be better to raise an error indicating that `particular` must be True when setting `quick` to True or False?

Reply:
I think perhaps it would be good to give a warning (not a deprecation warning but a general warning) if `quick` is used without `particular`. I often see people try various combinations of parameters in an attempt to make `solve` faster and I can imagine seeing a parameter like `quick` and trying that without expecting this behaviour.